File: dataset/human/python/file_5621.py
import math
s = str(input())
cou = 0
co = 0
ans = 0
while cou + co +2 <= len(s):
  if s[cou] == "x":
    if s[-1-co] == "x":
      cou += 1
      co += 1
    else:
      cou += 1
      ans += 1
  else:
    if s[-1-co] == "x":
      co += 1
      ans += 1
    else:
      if s[cou] == s[-1-co]:
        cou += 1
        co += 1
      else:
        print(-1)
        exit()
print(ans)

File: dataset/human/c/file_5621.c
#include<stdio.h>
#include<string.h>
#include<stdlib.h>
#include<math.h>

int main(){
    long long int x,ans=0,mod;
    scanf("%lld",&x);
    ans=((x-1)/11)*2+2;
    mod=x%11;
    if(mod<=6 &&mod>=1)ans--;
    printf("%lld",ans);
    
    return 0;
}
